src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -478,6 +476,51 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
         });
   }
 
+  /**
+   * Redirects to login page if feedback session is accessible.
+   *
+   * @param auth The authentication information used in the redirection URL.
+   */
+  redirectToLoginPageIfFeedbackSessionAccessible(auth: AuthInfo): void {
+    this.loadCourseInfo();
+    this.loadPersonName();
```

Comment:
Are these needed? This is an unhappy path after all.

Reply:
I agree that since this is an unhappy path, the above methods are unnecessary. I added them due to inconsistency as follows:

This is an unhappy path for an unregistered user:
<img width="960" alt="image" src="https://user-images.githubusercontent.com/75534016/169929337-ee241b67-88bd-4884-b73e-a7238cbfc7b9.png">

This is an unhappy path for a registered user (without the above methods):
<img width="960" alt="image" src="https://user-images.githubusercontent.com/75534016/169929282-1b8532bf-0a9f-4489-83e8-c2aff10deaac.png">

If such inconsistency is negligible, I'll proceed to remove these methods.

Reply:
Ah I see!